experiments/integrated/global/global_soil_canopy.jl

```diff
@@ -24,6 +24,11 @@ using Statistics
 using Dates
 import NCDatasets
 
+using ClimaDiagnostics
+using ClimaAnalysis
+import ClimaAnalysis.Visualize as viz
+using ClimaUtilities
```

Comment:
I would recommend using `import` instead of `using` to keep the namespace tidier